utils/test/event_validators/after_pectra.py

```diff
@@ -0,0 +1,104 @@
+from brownie.network.event import EventDict
+from utils.config import (
+    AGENT,
+)
+from utils.test.event_validators.permission import validate_grant_role_event, validate_revoke_role_event
+from utils.test.event_validators.common import validate_events_chain
+
+# Roles
+EXITED_VALIDATORS_PER_DAY_LIMIT_MANAGER_ROLE = "0x60b9982471bc0620c7b74959f48a86c55c92c11876fddc5b0b54d1ec47153e5d"
```

Comment:
Roles keccaks have to be moved to test_2025_05_21, as we don't check the parameters in helpers